### DISCUSSION TITLE
How do i use bowls in custom recipes?

### DISCUSSION THREAD

**Discussion by itsARprod** [vectorwing/FarmersDelight#964] [Q&A] [answered]
So I'm making an addon, and one item i want to be served in a bowl when made in a cooking pot. I haven't found anything online and ChatGPT just keeps making stuff up. If more info is needed, please ask :)

**Comment by vectorwing** [comment, marked as answer]
You can follow this page from our wiki:
https://github.com/vectorwing/FarmersDelight/wiki/Cooking-Pot-Recipes

As you're making an add-on, I imagine you have created the item in your output. In that case, you could set its crafting remainder as a bowl on the item properties, and the Cooking Pot will use it by default. For example:
`new Item.Properties().craftRemainder(Items.BOWL);`

Alternatively, for cooking recipes, you can define the `container` field, which will tell the Cooking Pot to treat this item as the serving container. This is used in vanilla soups and Stuffed Pumpkin, for example. However, it won't make the meal give that item back; it will only be required to collect a serving from the pot.
```
"container": {
    "item": "namespace:your_custom_container"
  },
```

**Reply by itsARprod** [reply]
Thanks! ChatGPT actually did the same thing but it accidentally put "container" inside of "result".
